api/CONTRIBUTING.md

```diff
@@ -39,7 +39,7 @@ Or to use a hermetic Docker container:
 ```
 
 This process builds RST documentation directly from the proto files, merges it with the static RST
-files, and then runs [Sphinx](https://www.sphinx-doc.org/en/stable/rest.html) over the entire tree to
+files, and then runs [Sphinx](https://www.sphinx-doc.org/en/master/index.html) over the entire tree to
```

Comment:
these links are not equivalent - im wondering what the benefit is - also we are tracking stable not master